GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+To indicate the conclusion of the modeling phase and that the model is ready for release and adoption, the modeling expert adds the label "MS2-Approved" to the PR.
+### MS3 Release Model
+The domain expert then checks with the use case team from which the initial modeling request originated to whether the proposed changes fulfill the requirements and that the use case will use it. There is a MS3 checklist in the PR template to guide the approval. This checklist, for example, asks whether there is at least one data provider and one data consumer planing to exchange data based on the definition in the proposed model. There is no specific process for getting the approval for the use case since this strongly depends on its working mode. However, one proven practice is to post the model and visualization in the working space of the use case team and ask for comments during a dedicated review period which could be one week. After the approval of the use case, the modeling team checks during one of its meetings that the proposed changes align with the other models in Tractus-X and that the model is ready for adoption from a modeling standpoint. The discussion of the model also helps the modeling team members to keep track of current modeling activities. If the use case and the modeling team approve the PR, the modeling team assigns the label "MS3-Approved" and merges the PR into the main branch of this Tractus-X models repository.
+
+## Labels
+To fascilitate the governance and keep track of ongoing PRs and issue in GitHub we use a set of labels as described in the following table: 
+
+| label | description |
+----- | -------
+Model Update | A request for updating an existing model
+New Model | A request for a new model
+Deprecate | A request for deprecating an existing model
+MS1-Approved | Checklist "MS1 Request for Model Developement" is approved.
+MS2-Approved | Checklist "MS2 Valid Model" is approved.
+MS3-Approved | Checklist "MS3 Release Model" is approved. The associated pull request can be merged to the "main-branch".
+Review Required | For this PR or issue a review by a modeling expert is requested.
+Modeling Team | This issue or PR should be discussed in the modeling team.
+
+## Retiring a model
+Once a model gets the status "DEPRECATED" it should not be used anymore by new use cases. Existing applications should look for a way to migrate to an alternative version or model. In alignment with the overall release cycles of Tractus-X, the modeling team tags the current main branch with the corresponding Tractus-X release and publishes modeling release notes. These release notes contain a hint about all model versions that changed the status to "DEPRECATED" since the previous Tractus-X release. The modeling team will delete these models two Tractus-X releases cycles later. So use cases are advised to react to the deprecation of models within the next release cycle.
```

Comment:
the status is DEPRECATE. I would prefer DEPRECATED but that's not how it is right now

Reply:
So I kept it for this PR, is it ok with you? I would adapt this later with a specific PR and agree that deprecated seems better.